src/main/java/example/model/Event.java

```diff
@@ -0,0 +1,17 @@
+package example.model;
+
+import jakarta.persistence.Column;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
+public class Event {
+
+    @Column(unique=true)
```

Comment:
this is a DTO object, Column is redundant here.

Reply:
Done